internal/backends/mcu/profiler.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+use crate::Devices;
+
+pub enum Timer {
+    #[cfg(debug_performance)]
+    Running { start_time: core::time::Duration },
+    #[cfg(debug_performance)]
+    Stopped { elapsed: core::time::Duration },
+    #[cfg(not(debug_performance))]
+    Noop,
+}
```

Comment:
```suggestion
#[cfg(slint_debug_performance)]
pub enum Timer {
    Running { start_time: core::time::Duration },
    Stopped { elapsed: core::time::Duration },
}

#[cfg(not(debug_performance))]
pub struct Timer {}
```

Should also simplify the implementation so there are less #[cfg]